Unity-Technologies/usd-unity-sdk
# [USDU-292] Adds testcase to check physical camera data on export and import

## Purpose of this PR

**Ticket/Jira #:**

[USDU-292](https://jira.unity3d.com/browse/USDU-292)

## Testing
ExportPhysicalCamera_RetainsPhysicalRelatedData
1. Adds Camera component to a gameObject
2. Set Physical Camera setting to True
3. Set focalLength, sensorSize and lensShift
4. Export the camera object
5. Ensure the exported camera object has physical camera data within its USD properties

ImportPhysicalCamera_PhysicalDataKept
1. Created a Physical camera object on Blender and exported as USDA file (physicalCam.usda)
2. Do Assertion on physical camera properties set in usda file

**Complexity:**
1

**Halo Effect:**
1

## Additional information

**Note to reviewers:**

**Reminder:**
